src/components/proofEditor/issueInformation.tsx

```diff
@@ -7,14 +8,18 @@ export interface Props {
 	issue: Issue;
 }
 
-export class IssueInformation extends React.Component<Props> {
+export class IssueInformation extends React.Component<Props, {}> {
 
 	public render() {
 
 		let displayedPosition: any = "";
 		const position = this.props.issue.position;
 		if (position) {
```

Comment:
`position` is a weird name for a boolean. Also the fact that you set it equal to a position of an issue is confusing. Additionally it is not obvious for me what "props" stands for. Properties maybe? If so: Properties of what?

Reply:
I forgot to address the props question.
`props` is the standard attribute of a react component. So it should be clear when I pass the interface with the name Props. When you pass the interface into React.Component<Props, State> like this then Props will become the definition of the props and State of the states. In this case we don't use states (so I could have and should have built a stateless component) so therefore it's just {} for the State.

When you create a react component you will use them in a different location like this:

<ComponentName />

But you can also pass information to it like this.
<ComponentName value="Some String" />

You can then access this information value like via:
`this.props.value`

The interface that is passed is for Typescript to have a description of the props. It just mirrors the internal attribute name.